cost-analyzer/templates/controller-secrets.yaml

```
@@ -0,0 +1,14 @@
+{{- if (.Values.clusterController).enabled }}
+{{- if .Values.clusterController.createClusterControllerSecret }}
```

```suggestion
{{- if and .Values.clusterController.createClusterControllerSecret .Values.clusterController.kubecostAPIKey }}
```

We should only be creating this secret, if the user has provided a kubecostAPIKey